.test-infra/dockerized-jenkins/plugins.txt

```diff
@@ -22,7 +22,6 @@ throttle-concurrents
 JiraTestResultReporter
 
 active-directory
-analysis-core
```

Comment:
I don't have much context here, but should we be replacing these with their [newer counterparts](https://github.com/jenkinsci/analysis-core-plugin/tree/master#this-plugin-reached-end-of-life)?